src/object-diagram/reader/panelViewInput.ts

```diff
@@ -0,0 +1,114 @@
+import { PanelViewInput, VariableRelation } from '../../model/panelViewInput';
```

Comment:
The name of this file is misleading. `panelViewInputObjectDiagramReader.ts` is totally fine.

Reply:
* All file and exported class names are now synchronized
* Index files are only used for re-exports (currently none)